pkg/handlers/webhook/webhook.go

```diff
@@ -66,13 +69,36 @@ type EventMeta struct {
 // Init prepares Webhook configuration
 func (m *Webhook) Init(c *config.Config) error {
 	url := c.Handler.Webhook.Url
+	cert := c.Handler.Webhook.Cert
+	tlsSkip := c.Handler.Webhook.TlsSkip
 
 	if url == "" {
 		url = os.Getenv("KW_WEBHOOK_URL")
 	}
+	if cert == "" {
+		cert = os.Getenv("KW_WEBHOOK_CERT")
```

Comment:
I would add a comment on that env var option in the upper section, next to "exportKW_WEBHOOK_URL=webhook_url"

Reply:
I cannot understand what this comment mean, could you explain more?

Reply:
Sorry for not being clear enough.
At the beginning of this file (line 39), there's a section that document the option to add the "KW_WEBHOOK_URL" environment variable.
I suggested to add explanation about the new environment option, "KW_WEBHOOK_CERT ", as well

Reply:
Sorry for not being clear enough.
At the beginning of this file (line 39), there's a section that document the option to add the "KW_WEBHOOK_URL" environment variable.
I suggested to add explanation about the new environment option, "KW_WEBHOOK_CERT ", as well

Reply:
Thanks for detail. I fixed webhookErrMsg according to your comment, please check it.